src/components/BalanceService.tsx

```diff
@@ -14,9 +15,11 @@ class BalanceService {
     constructor(memoryService: MemoryService) {
         this.memoryService = MemoryService.instance
         this.balances = [];
+        makeAutoObservable(this)
     };
 
     static instance = new BalanceService(MemoryService.instance);
 };
+export const myBalance = new BalanceService()
```

Comment:
instead of exporting myBalance, use BalanceService.instance.

Reply:
ok
done